cookbooks/child-workflows/child-workflows-megafood.md

```diff
@@ -0,0 +1,321 @@
+# Instafood: Cadence Child Workflow Cookbook
+
+![Instaclustr Managed Cadence](images/Instaclustr_Product_Managed_Cadence.png)
+
+## Introduction
+
+### Who is this cookbook for?
+
+This cookbook is for developers and engineers of all levels looking to understand how child workflows work in Cadence. The
+recipe in this book provides *"Hello World!"* type examples based on simple scenarios and use cases.
+
+### What you will learn
+
+How to setup a simple Cadence application which implements child workflows on Instaclustr's Managed Service Platform.
+
+### What you will need
+
+- An account on Instaclustr’s managed service platform (sign up for a free trial using the
+  following [signup link](https://console2.instaclustr.com/signup))
+- Basic Java 11 and Gradle installation
+- IntelliJ Community Edition, Visual Studio Code or any other IDE with Gradle support
+- Docker (optional: only needed to run Cadence command line client)
+
+### What is Cadence?
+
+A large number of use cases span beyond a single request-reply, require tracking of a complex state, respond to asynchronous events, and communicate to external unreliable dependencies. The usual approach to building such applications is a hodgepodge of stateless services, databases, cron jobs, and queuing systems. This negatively impacts developer productivity as most of the code is dedicated to plumbing, obscuring the actual business logic behind a myriad of low-level details.
+
+Cadence is an orchestration framework that helps developers write fault-tolerant, long-running applications, also known as workflows. In essence, it provides a durable virtual memory that is not linked to a specific process, and preserves the full application state, including function stacks, with local variables across all sorts of host and software failures. This allows you to write code using the full power of a programming language while Cadence takes care of durability, availability, and scalability of the application.
+
+### What are child workflows?
+
+Cadence's core abstraction is a fault-oblivious stateful **workflow**. A workflow may start other workflows and wait for them to complete, either synchronously or asynchronously. These are known as **child workflows**, and are a powerful tool which enable developers to coordinate large and complex tasks within the Cadence framework.
+
+Workflows evolve over time and child workflows may develop as a result of that evolutuion, such as when two existing workflow processes are joined, or designed from the start to take advantage the features available in Cadence.
+
+![Child workflow diagram](images/TBA.png)
+
+*Fig 1. Flow diagram for a child workflow process*
+
+### What are the advantages of child workflows?
+
+Child workflows have similar functionality to **activities**. An activity is a business-level function that implements application logic such as calling a service or transcoding a media file. 
```

Comment:
It's mentioned below that activities are executed by workflows (and that there are limits), but here they are mentioned but not explained how they tie in with workflows. Maybe include verbatim the wording from the other cookbook:

> In order to fulfill deterministic execution requirements, workflows are not allowed to call any external API directly. Instead they orchestrate execution of activities. An activity is business-level function that implements application logic such as calling a service or transcoding a media file. Cadence does not recover activity state in case of failures, therefore an activity function is allowed to contain any code without restrictions.